src/DipoleAlgorithms.cc

```diff
@@ -137,34 +137,54 @@ void NaiveDipoleCentroid::measure(
 ) const {
     afw::detection::PeakCatalog const& peaks = source.getFootprint()->getPeaks();
 
-    naiveCentroid(source, exposure, peaks[0].getI(), (peaks[0].getPeakValue() >= 0 ?
-                                                       getPositiveKeys() :
-                                                       getNegativeKeys()));
-    if (peaks.size() > 1) {
-        naiveCentroid(source, exposure, peaks[1].getI(), (peaks[1].getPeakValue() >= 0 ?
-                                                           getPositiveKeys() :
-                                                           getNegativeKeys()));
+    /*
```

Comment:
Sorry I didn't get a chance to look carefully before this is rewritten.  Converting the debug logging in C++ to use `lsst.log` is still work in progress so I don't have a good example in a stable place yet, but if you don't mind, this might be an example: https://github.com/lsst/afw/blob/4afd3c77e952ac24efa75f5ac87e5d04f2f4ebe4/src/detection/PsfFormatter.cc    (No promise this commit will stay long, sorry!)

Get a logger `LOG_LOGGER _log = LOG_GET("ip.diffim.DipoleAlgorithms");` and then log to it  `LOGL_DEBUG(_log, "messages")`     For more details, please see [RFC-203](https://jira.lsstcorp.org/browse/RFC-203?focusedCommentId=52671&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-52671) and later those descriptions will be added to some formal documentation.